packages/cli/src/commands/doctor/healthchecks/cocoaPods.ts

```diff
@@ -1,11 +1,92 @@
+import execa from 'execa';
+import chalk from 'chalk';
+import readline from 'readline';
+import wcwidth from 'wcwidth';
+import {logger} from '@react-native-community/cli-tools';
 import {checkSoftwareInstalled} from '../checkInstallation';
-import {installCocoaPods} from '../../../tools/installPods';
+import {
+  promptCocoaPodsInstallationQuestion,
+  runSudo,
+} from '../../../tools/installPods';
+import {brewInstall} from '../../../tools/brewInstall';
 import {HealthCheckInterface} from '../types';
 
+function calculateQuestionSize(promptQuestion: string) {
+  return Math.max(
+    1,
+    Math.ceil(
+      wcwidth(promptQuestion) /
+        (process.stdout.columns! ? process.stdout.columns! : 80),
```

Comment:
Not quite sure why you have to force unwrap here. Can we do something like this instead:
```suggestion
        (process.stdout.columns || 80),
```

Reply:
I have tried it but TypeScript gives an error, I'll check it again.

I do agree that this piece of code is a little bit strange.

Reply:
Aaaand... I just tried this now and don't get an error 😐

Just pushed it, thanks for the hint!